pkg/controller/seed-controller-manager/default-application-controller/controller.go

```diff
@@ -355,6 +380,13 @@ func ApplicationInstallationReconciler(ctx context.Context, logger *zap.SugaredL
 				app.Spec.Values = *application.Spec.DefaultValues
 			}
 
+			// add reconciliation interval to the system apps except Cilium, as it is managed by its own controller.
+			if app.Labels[appskubermaticv1.ApplicationManagedByLabel] == appskubermaticv1.ApplicationManagedByKKPValue {
+				if app.Labels[appskubermaticv1.ApplicationTypeLabel] != appskubermaticv1.ApplicationTypeCNIValue {
+					app.Spec.ReconciliationInterval = reconciliationInterval
+				}
+			}
```

Comment:
Why do we need a reconciliation interval here? This will only work for cluster-autoscaler application when it's either being defaulted or enforced. This is not how this application is installed most of the time.

Reply:
If you want to set `ReconciliationInterval` for some apps in general i'd rather suggest adding `DefaultReconciliationInterval` in ApplicationDefinition and then if the value is configured(not empty/null) use it in the ApplicationInstallation,